spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/Issue2379Test.java

```diff
@@ -0,0 +1,29 @@
+package edu.umd.cs.findbugs.detect;
+
+import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+import edu.umd.cs.findbugs.AbstractIntegrationTest;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
+
+
+public class Issue2379Test extends AbstractIntegrationTest {
+    @Test
+    public void test() {
+        performAnalysis("ghIssues/Issue2379.class");
+
+        BugInstanceMatcher bugTypeMatcher = new BugInstanceMatcherBuilder()
+                .bugType("UPM_UNCALLED_PRIVATE_METHOD")
+                .inMethod("unusedValues")
+                .build();
+
+        // Check that we've found the uncalled private method
+        assertThat(getBugCollection(), containsExactly(1, bugTypeMatcher));
+
+        // Check that it was the only bug we've found
+        assertThat(getBugCollection(), containsExactly(1, new BugInstanceMatcherBuilder().build()));
```

Comment:
Can you please add the bugtype to the `BugInstanceMatcher`? If later a new detector is added, which finds a bug in the test class, this test would fail, which is not intended.